CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Added 🚀
 
+-   Show file count of folders in Attribute-Side-Bar
```

Comment:
Please include the issue number next to the change, also for added features we decided to try for a bit more elaborate texts to help users catch up.

Reply:
What do you think about adding the quick screenshot from PR description?

Reply:
And it is added :)

Reply:
Good idea, we might use that from now on